web/docs/pkgconfig.md

```diff
@@ -0,0 +1,102 @@
+# pkg-config
+
+We default to [pkgconf](https://github.com/pkgconf/pkgconf) as our pkg-config
+implementation. This page lists some Windows specific topics/issues/challenges
+for working with pkg-config on Windows.
+
+## Environment Variables
+
+By default, the MSYS2 shells set the following environment variables with paths for the active environment to support pkgconf:
+
+* `PKG_CONFIG_PATH` - e.g. `/ucrt64/lib/pkgconfig:/ucrt64/share/pkgconfig`
+* `PKG_CONFIG_SYSTEM_INCLUDE_PATH` - e.g. `/ucrt64/include`
+* `PKG_CONFIG_SYSTEM_LIBRARY_PATH` - e.g. `/ucrt64/lib`
+
+## Prefix / Relocation
+
+The first few lines of a .pc file usually look something like this:
+
+```bash
+prefix=/ucrt64
+includedir=${prefix}/include
+libdir=${prefix}/lib
+```
+
+As you can see `/ucrt64` is not a proper Windows path, but that's not a problem
+because by default `prefix` will be ignored, or rather re-defined by
+pkgconf/pkg-config based on the location of the .pc file itself. It will strip
+of `/<...>/pkgconf` from the directory of the `.py` file and use the resulting
```

Comment:
```suggestion
of `/<...>/pkgconfig` from the directory of the `.pc` file and use the resulting
```

?

Reply:
thanks

Reply:
oops, just noticed that line probably should have started with "off" rather than "of" too

Reply:
thanks, fixed